notebooks/en/rag_prompt_injection_defense_with_chainguard.ipynb

```diff
@@ -0,0 +1,526 @@
+{
```

Comment:
<p>Can you replace this with an open-source model instead? It can also be a hosted endpoint at Hugging Face for instance but with an open-source alternative.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/78/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>